lib/checker.js

```diff
@@ -45,12 +45,62 @@ Checker.prototype.checkFile = function(path) {
 };
 
 /**
+ * Fixes single file.
+ *
+ * @param {String} path
+ * @returns {Promise * Errors}
+ */
+Checker.prototype.fixFile = function(path) {
+    if (!this._configuration.isFileExcluded(path)) {
```

Comment:
Non-blocker: This style becomes pervasive throughout community contributions. What about opting for an early return?

```js
if (this._configuration.isFileExcluded(path)) {
  return Vow.resolve(null);
}

return vowFs.read(path, ...
```

The idea is along the lines of: http://sourcemaking.com/refactoring/replace-nested-conditional-with-guard-clauses